### PR TITLE
Clicking on the current image cycles to the next one (And new have-stream images)

### PR DESCRIPTION
**Pull request by tralezab**
## What

Makes OnClick use `selectNextImage` instead of `selectRandomImage`, which as the name implies, selects the next image in the set instead of randomly choosing.

Adds love.png for having a stream.

## Why

Without seeing duplicates you really get to enjoy the full gallery, and the have-stream imageset only had a single image.

### CODE REVIEW

**Review by saplinganon — CHANGES_REQUESTED** [review]
Sounds like a good idea. However, I'd like the cycling order to be randomized as well if we're going to implement this.

I also want to restrict the I Don't Miss Fauna set to just normal images (thus implying that saplings only get weird when she's gone).

**Comment by tralezab** [comment]
> Sounds like a good idea. However, I'd like the cycling order to be randomized as well if we're going to implement this.

Sure, should just be a shuffle after the imageset is randomized for the first time.

> I also want to restrict the I Don't Miss Fauna set to just normal images (thus implying that saplings only get weird when she's gone).

I like that idea a lot actually. Keeping love, removing the deformed version.

File: pages/index.js [hunk 1]
```diff
@@ -5,17 +5,18 @@ import { ERROR_IMAGE_SET, HAVE_STREAM_IMAGE_SET, NO_STREAM_IMAGE_SET } from "../
...
+function selectRandomImage(fromSet) {
     return fromSet[(Math.random() * fromSet.length) | 0]
 }
 
+/**
+ * Returns the next image in an imageset, wrapping around to the start after reaching the end of the array.
+ */
+function selectNextImage(fromSet, currentImage) {
+    let nextIndex = fromSet.indexOf(currentImage) + 1
+    return fromSet[(((nextIndex - 0) % (fromSet.length - 0)) + 0)  | 0]
```
[the remark this answers is not in the document]
Comment (tralezab) [reply pages/index.js:17]: Er...  whoops! It's surprising how simple wrapping gets when your lower bound is zero. I should have paid more attention.

**Comment by tralezab** [comment]
Alright tell me if I did anything bad with state there, from testing this works nicely for shuffling the imageset

**Review by saplinganon — CHANGES_REQUESTED** [review]

**Comment by saplinganon** [comment]
Live now, thanks for the PR!

**Pull request merged**